OpenAnatomyExport/OpenAnatomyExport.py

```diff
@@ -497,6 +535,45 @@ def addModelToRenderer(self, inputModelNode, outputModelNode, boostGouraudColor=
 
     return True
 
+  def createPlaneModelFromMarkupsPlane(self,planeMarkup):
+    cornerPoints = vtk.vtkPoints()
+    planeMarkup.GetPlaneCornerPoints(cornerPoints)
+
+    from vtk.util.numpy_support import vtk_to_numpy
+    cornerPoints_np = vtk_to_numpy(cornerPoints.GetData())
+
+    import numpy as np
+    def sort_points(pts):
```

Comment:
No point sorting is needed. Use the plane bounds and object-to-world transform instead:

```
planeMarkup = getNode('P')

planeBounds = planeMarkup.GetPlaneBounds()
objectToWorld = vtk.vtkMatrix4x4()
planeMarkup.GetObjectToWorldMatrix(objectToWorld)

planeSource = vtk.vtkPlaneSource()
planeSource.SetOrigin(objectToWorld.MultiplyPoint([planeBounds[0], planeBounds[2], 0.0, 1.0])[0:3])
planeSource.SetPoint1(objectToWorld.MultiplyPoint([planeBounds[1], planeBounds[2], 0.0, 1.0])[0:3])
planeSource.SetPoint2(objectToWorld.MultiplyPoint([planeBounds[0], planeBounds[3], 0.0, 1.0])[0:3])
planeModel = slicer.modules.models.logic().AddModel(planeSource.GetOutputPort())
```